tests/conftest.py

```diff
@@ -238,3 +239,38 @@ def unprivileged_client(
 
         else:
             yield admin_client
+
+
+@pytest.fixture(scope="session")
+def dsc_resource(admin_client: DynamicClient, name: str = "default-dsc"):
```

Comment:
name: str = "default-dsc" - you should use `request` and then `request.param` to control the name
i think that this should be set in global_config.py

Reply:
Rather than use request/request.param, I think this is a good candidate for using py_config. The name of the DSC is known and stable, and if we need to update it in some specific scenario we can override it like we do for applications_namespace